src/srt-stream-writable.js

```diff
@@ -24,6 +24,14 @@ class SRTWriteStream extends Writable {
     this.fd = null;
   }
 
+  stats(clear) {
```

Comment:
are you using this writable-stream class in your use-case?

just asking because it is not using the async-API, so it can get quite limited in performance, i.e potentially cause main-loop congestion or blocking depending on the libSRT interactions